lib/WindowsCompat.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  pathToCanonical(path: string): string {
+    return path.replace(/(^[A-Z]:\\|\\)/g, '/');
```

Comment:
If we get a backslash in path I guess it must be escaped right?
So we'd just need to avoid replacing double-backslashes?